PopupExample/PopupExample/ContentView.swift

```diff
@@ -284,11 +284,14 @@ struct ContentView : View {
                 ActionSheetSecond()
             } customize: {
                 $0
-                    .type(.toast)
+                    .type(.scroll)
+                    .scrollViewColor(.white)
+                    .scrollViewHeaderView { scrollViewHeader() }
```

Comment:
.scrollViewColor(.white)
.scrollViewHeaderView { scrollViewHeader() }

these two make no sense for non scroll-type popups, right? so they should be the arguments of the enum case .scroll